addons/armaos/functions/fnc_computer_addUser.sqf

```diff
@@ -0,0 +1,51 @@
+/**
+ * PUBLIC
+ *
+ * Adds a user to a given computer by providing username and password. Also a user directory will be created.
+ *
+ * Arguments:
+ * 1: Computer <OBJECT>
+ * 2: Username <STRING>
+ * 3: Password <STRING>
+ *
+ * Results:
+ * None
+ *
+ * Example:
+ * [_computer, "admin", "admin123"] call AE3_armaos_fnc_computer_addUser;
+ *
+ */
+
+params ["_computer", "_username", "_password"];
+
+// Get userlist and filesystem from computer
+private _userlist = _computer getVariable ["AE3_Userlist", createHashMap];
+private _filesystem = _computer getVariable ["AE3_filesystem", []];
+
+// Add user to userlist
+_userlist set [_username, _password];
+
+// Add user directory in /home/
+if(!(_username isEqualTo "root")) then
+{
+    try
+    {
+        [[], _filesystem, "/home/" + _username, "root", _username] call AE3_filesystem_fnc_createDir;
+    } 
+    catch
+    {
+        private _normalizedException = _exception regexReplace ["'(.+)'", "'%1'"];
+        if (_normalizedException isEqualTo (localize "STR_AE3_Filesystem_Exception_AlreadyExists")) then
+        {
+            diag_log format ["AE3 exception: %1", _exception];
```

Comment:
Yes, that's a good idea. I will also implement that for addFile and addDir. It's the same case. This error message will trigger if placed in Eden Editor and also if placed in Zeus. I also needed to change function priorities for modules. The execution order for modules is defined by a priority. With my adjustment execution order is 1. addUser 2. addDir 3. addFile This will prevent an error when you create a user by module and also create a file by module that is placed inside the users home directory.